pkg/convert/json_nglocalize.go

```diff
@@ -66,7 +68,19 @@ func ToNgLocalize(translation model.Translation) ([]byte, error) {
 	}
 
 	for _, msg := range translation.Messages {
-		ng.Translations[msg.ID] = "" // TODO: convert msg.Message from MF2 format.
+		tree, err := ast.Parse(msg.Message)
+		if err != nil {
+			return nil, fmt.Errorf("parse mf2 message: %w", err)
+		}
+
+		switch mf2Msg := tree.Message.(type) {
+		case nil:
+			ng.Translations[msg.ID] = ""
+		case ast.SimpleMessage:
+			ng.Translations[msg.ID] = PatternsToMsg(mf2Msg)
```

Comment:
I think this should be unexported
```suggestion
			ng.Translations[msg.ID] = patternsToMsg(mf2Msg)
```